feature-toggles/feature-toggles-impl/src/test/java/com/duckduckgo/feature/toggles/codegen/ContributesRemoteFeatureCodeGeneratorBucketAssignmentTest.kt

```diff
@@ -35,8 +35,8 @@ import org.mockito.kotlin.mock
 import org.mockito.kotlin.whenever
 import org.robolectric.ParameterizedRobolectricTestRunner
 
-private var enables = 1
-private var disables = 1
+private var enables = 0
+private var disables = 0
```

Comment:
@aitorvs initializing these to 1 seems wrong but I could be misunderstanding why it's like that. can you check if 0 makes sense?